components/infobox/wikis/valorant/infobox_map_custom.lua

```diff
@@ -0,0 +1,72 @@
+---
+-- @Liquipedia
+-- wiki=valorant
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector')
+local Map = Lua.import('Module:Infobox/Map')
+local Flags = Lua.import('Module:Flags')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+---@class ValorantMapInfobox: MapInfobox
+local CustomMap = Class.new(Map)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomMap.run(frame)
+	local map = CustomMap(frame)
+	map:setWidgetInjector(CustomInjector(map))
+
+	return map:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local args = self.caller.args
+
+	if id == 'location' then
+		return {
+			Cell{
+				name = 'Location',
+				content = {Flags.Icon{flag = args.country, shouldLink = false} .. '&nbsp;' .. args.country},
+			},
+		}
+	elseif id == 'custom' then
+		Array.appendWith(
+			widgets,
+			Cell{name = 'Bomb Sites', content = {args.bombsites}},
+			Cell{name = 'Teleporters', content = {args.teleporters}},
+			Cell{name = 'Game Mode', content = {args.mode}}
```

Comment:
Same with game mode

Reply:
Game mode is a new field. It is being added as the game only recently made map game modes relevant. There should be no usage outside of the bind map. I do have icons for the game modes, but do not yet have them uploaded, so that will be in a future PR (I might need to edit the icons which will take some time, thus why im not including them here)